projects/canopy/src/lib/side-nav/side-nav.stories.ts

```diff
@@ -0,0 +1,103 @@
+import { Component, Input } from '@angular/core';
+
+import { object, withKnobs } from '@storybook/addon-knobs';
+import { moduleMetadata } from '@storybook/angular';
+
+import { LgSideNavModule } from './side-nav.module';
+import { notes } from './side-nav.notes';
+import { LgButtonModule } from '../button';
+import { SideNavBarItem } from './side-nav.interface';
+import { LgMarginModule } from '../spacing';
+
+@Component({
+  selector: 'story-side-nav',
+  template: `
+    <lg-side-nav>
+      <lg-side-nav-bar label="Side Nav Demo">
+        <a
+          *ngFor="let item of navItems; index as i"
+          class="lg-side-nav-bar-item__container"
+          id="side-nav-{{ i }}"
+          href=""
+          (click)="onClick($event, i)"
+        >
+          <lg-side-nav-bar-item [isActive]="i == selectedNavIndex">
+            <lg-side-nav-bar-item-heading>{{ item.title }}</lg-side-nav-bar-item-heading>
+            <lg-side-nav-bar-item-content *ngIf="item.description">{{
+              item.description
+            }}</lg-side-nav-bar-item-content>
+          </lg-side-nav-bar-item>
+        </a>
+        <lg-side-nav-bar-footer>
+          <a
+            lg-button
+            lgMarginTop="xxl"
+            variant="outline-primary"
+            [fullWidth]="false"
+            href="#"
+            >Logout</a
+          >
+        </lg-side-nav-bar-footer>
+      </lg-side-nav-bar>
+      <lg-side-nav-content></lg-side-nav-content>
+    </lg-side-nav>
+  `,
+})
+export class StorySideNavComponent {
+  selectedNavIndex = 0;
+
+  @Input()
+  navItems = [];
+
+  onClick = (ev, ix) => {
+    ev.preventDefault();
+    this.selectedNavIndex = ix;
+  };
+}
+
+export default {
+  title: 'Components/Side Nav',
+  excludeStories: ['StorySideNavComponent'],
+  parameters: {
+    decorators: [
+      withKnobs,
+      moduleMetadata({
+        declarations: [StorySideNavComponent],
+        imports: [LgSideNavModule, LgButtonModule, LgMarginModule],
+      }),
+    ],
+    notes: {
+      markdown: notes,
+    },
+  },
+};
+
+const getDefaultNavItems = (): Array<SideNavBarItem> => [
```

Comment:
It might be an idea to make these items more generic and less L&G specific?

Reply:
Yep I think that would better, I'm not a Lorem Ipsum fan but it might keep it more generic.

Reply:
Fixed.